esi_leap/db/sqlalchemy/api.py

```diff
@@ -118,11 +118,15 @@ def offer_get_all(filters):
     start = filters.pop('start_time', None)
     end = filters.pop('end_time', None)
     time_filter_type = filters.pop('time_filter_type', None)
+    status = filters.pop('status', None)
     a_start = filters.pop('available_start_time', None)
     a_end = filters.pop('available_end_time', None)
 
     query = query.filter_by(**filters)
 
+    if status:
+        query = query.filter((models.Offer.status.in_(status)))
```

Comment:
Is there any particular reason for the extra brackets here?

Reply:
Ah, I just did the way it is done in method lease_get_all() line number 293 in esi_leap/db/sqlalchemy/api.py

Reply:
Ah gotcha. It probably doesn't need to be done that way in `lease_get_all()` but for the sake of consistency I suppose it makes sense to keep it like this then.

Reply:
Yeah!